### PR TITLE
UINV-187 Update invoice view pane

### PR DESCRIPTION
**Pull request by AndreiShumski2**
## Purpose
https://issues.folio.org/browse/UINV-187

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Comment by aliaksei-chumakou** [comment]
@AndreiShumski2 you still have the Description column, what is the reason to move `icon="exclamation-circle"` ?

**Comment by AndreiShumski2** [comment]
> @AndreiShumski2 you still have the Description column, what is the reason to move `icon="exclamation-circle"` ?

I was thinking that this icon doesn't belong to the 'Description' column and it should appear at the line start.

File: src/invoices/InvoiceDetails/InvoiceLines/InvoiceLines.js [hunk 1]
```diff
@@ -33,22 +50,7 @@ const InvoiceLines = ({
...
-    total: ({ total }) => (
-      <AmountWithCurrencyField
-        amount={total}
-        currency={currency}
-      />
-    ),
-    arrow: () => <Icon icon="caret-right" />,
-    // eslint-disable-next-line react/prop-types
-    description: ({ description, poLineId }) => {
+    lineNumber: ({ id, poLineId }) => {
```
Comment (aliaksei-chumakou) [inline src/invoices/InvoiceDetails/InvoiceLines/InvoiceLines.js:53]: there is a `rowIndex` prop.

File: src/invoices/InvoiceDetails/InvoiceLines/InvoiceLines.js [hunk 2]
```diff
@@ -64,11 +66,36 @@ const InvoiceLines = ({
               &nbsp;
             </>
           )}
-          {description}
+          {findIndex(invoiceLinesItems, ['id', id]) + 1}
```
Comment (aliaksei-chumakou) [inline src/invoices/InvoiceDetails/InvoiceLines/InvoiceLines.js:69]: you can use `rowIndex` prop

**Review by NikitaSedyx — APPROVED** [review]

**Review by aliaksei-chumakou — APPROVED** [review]

File: translations/ui-invoice/en.json [hunk 3]
```diff
@@ -221,11 +221,15 @@
 
   "invoice.details.lines.title": "Invoice lines",
   "invoice.details.lines.add": "Add",
-  "invoice.details.lines.list.description": "Title",
-  "invoice.details.lines.list.number": "Line number",
-  "invoice.details.lines.list.quantity": "Quantity",
   "invoice.details.lines.list.adjustments": "Adjustments",
+  "invoice.details.lines.list.description": "Description",
```
Comment (AndreiShumski2) [inline translations/ui-invoice/en.json:225]: @dltj , Plz update translation for this key.

**Comment by dltj** [comment]
I had to wait until the end of the day because a new language was being added.  (Adding a new language touches every `stripes-*` and `ui-*` repo, so it backs up the CI jobs.)  Done in #355.

**Pull request merged**